lesson13/style.css

```diff
@@ -0,0 +1,97 @@
+:root {
+  font-family: Inter, Avenir, Helvetica, Arial, sans-serif;
+  font-size: 16px;
+  line-height: 24px;
+  font-weight: 400;
+
+  color-scheme: light dark;
+  color: rgba(255, 255, 255, 0.87);
+  background-color: #242424;
+
+  font-synthesis: none;
+  text-rendering: optimizeLegibility;
+  -webkit-font-smoothing: antialiased;
+  -moz-osx-font-smoothing: grayscale;
+  -webkit-text-size-adjust: 100%;
+}
```

Comment:
Q. Is this read in somewhere?

Reply:
Thank you for point that out, I did not notice that default files were there, I deleted unnecessary files. 
Please see the latest commit 1117e22f84f53bb2e3620a6a3b3020db03ed2978